libs/cart/state/src/reducers/cart-order/cart-order-state.interface.ts

```diff
@@ -1,8 +1,8 @@
-import { DaffLoadingState, DaffStateError } from '@daffodil/core/state';
+import { DaffStates, DaffStateError } from '@daffodil/core/state';
 import { DaffCartOrderResult } from '@daffodil/cart';
 
 export interface DaffCartOrderReducerState<T extends DaffCartOrderResult = DaffCartOrderResult> {
   cartOrderResult: T;
-  loading: DaffLoadingState;
+  loading: DaffStates;
```

Comment:
This can still be typed as a loading mutable state, right?

Reply:
Yes.